ethclient/ethclient.go

```diff
@@ -282,6 +282,9 @@ func toBlockNumArg(number *big.Int) string {
 	if number == nil {
 		return "latest"
 	}
+	if number.Sign() < 0 {
```

Comment:
I think we should be a bit more strict here and only allow -1 to mean pending.
That way, we can use other negative numbers for different things later.